### DISCUSSION TITLE
Let me buy you a coffee

### DISCUSSION THREAD

**Discussion by stnokott** [kinghat/tabbed-card#21] [General]
Heya,
not really an issue, just wanted to let you know I love this card!
I was looking for a way to show my apprecation in a materialistic way, but couldn't find any donation link similar to https://www.buymeacoffee.com.
Maybe you could add one?
Otherwise, you'd have to settle with mere words as thanks 😉

**Comment by kinghat** [comment]
thanks for your interest in the card. i never intended to be compensated in any way for this card, only to "scratch an itch." im not opposed to tips. i will probably add a `ko-fi` and `xmr` address for those that are inclined. if neither of those work for you, thats fine too. just enjoy the card :pray:

**Reply by kinghat** [reply]
added `ko-fi`: https://github.com/kinghat/tabbed-card/pull/24

**Reply by stnokott** [reply]
No intention of being compensated doesn't mean you can't still be compensated for the work other people profit from.
Thanks for adding.
